package.json

```diff
@@ -383,6 +383,29 @@
                     "default": "TargetLanguageCode",
                     "description": "Specifies which target language to use by default in the XLIFF Sync snippets.",
                     "scope": "resource"
+                },
+                "xliffSync.defaultLanguages": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "pattern": "^[a-z]{2}-[A-Z]{2}$"
```

Comment:
Is there a pattern that works for all? Or is it the best to just remove the pattern entirely?

Reply:
They are RFC 4646 or RFC 5646 language tags, but I think it might be better to just remove the pattern as I otherwise a RegEx that is a bit too complex. I think it's good enough if people can type in anything they want and think for themselves whether it is a valid tag. ;)